bibtex-completion.el

```diff
@@ -1321,6 +1379,13 @@ Self-contained means that cross-referenced entries are merged."
              concat
              (format "  %s = {%s},\n" name value)))))
 
+(defun bibtex-completion-add-pdf-attachment ()
+  ; this is basically a lowercase command alias for the function
+  ; ideally, that name should be deprecated in favor of this one
+  "Attach the PDFs of the entries with the given KEYS where available."
+  (interactive (list (bibtex-completion--read)))
+  (bibtex-completion-add-PDF-attachment))
```

Comment:
There's an interactive argument here but the function doesn't accept any arguments :?

Reply:
Oops; fixed.

Thanks.